src/main/java/ladder/domain/Player.java

```diff
@@ -0,0 +1,22 @@
+package ladder.domain;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Player {
+    private static final String SEPARATOR = ",";
+    private final List<Name> players;
+
+    public Player(String names) {
+        players = Arrays.stream(names.split(SEPARATOR))
+            .map(String::trim)
+            .map(Name::new)
```

Comment:
```suggestion
            .map(Name::new)
```
Name에서 처리해주도록 하면 어떨까요?